### ISSUE TITLE
Webpack 4 compatibility

### ISSUE THREAD

**Issue by IanVS** [theverything/webpack-create-file-plugin#1]
Hi, this is a useful plugin for me, due to the same use case as the poster in https://stackoverflow.com/questions/46172596/using-extracttextplugin-for-webpack-how-do-you-not-extract-css-and-create-an-em.  

However, I'm using webpack 4, and after adding this plugin, I'm now seeing 

```
DeprecationWarning: Tapable.plugin is deprecated. Use new API on `.hooks` instead
```

which I assume is because this plugin is not updated to the webpack 4 API.  Any chance you might consider supporting it?

**Comment by theverything** [comment]
Sure I'll take a look and update.

**Comment by theverything** [comment]
@IanVS https://github.com/theverything/webpack-create-file-plugin/commit/7943169b1015612fdba771d8f4d1a524a3f42763 should remove the deprecation warning. 

Published v1.0.0 to npm.

**Comment by IanVS** [comment]
That was fast, thanks!
